platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImpl.java

```diff
@@ -101,6 +101,7 @@ public ImportMigrationManagerImpl(
       // add a system contexts
       contexts.put(null, new ImportMigrationContextImpl(report, zip));
       zip.stream()
+          .filter(ze -> !ze.isDirectory())
```

Comment:
❓ Shouldn't we have a unit test for this?

Reply:
done